src/components/Table.tsx

```diff
@@ -50,6 +53,7 @@ const BaseTable = styled.table`
 	> tbody {
 		> tr {
 			> td {
+				text-align: left;
 				font-size: 16px;
```

Comment:
Out of context but isn't that supposed to be 14px in the body?

Reply:
Good spot!